src/TLuaInterpreter.cpp

```diff
@@ -10791,6 +10807,10 @@ int TLuaInterpreter::downloadFile(lua_State* L)
     host.updateProxySettings(host.mLuaInterpreter.mpFileDownloader);
     QNetworkReply* reply = host.mLuaInterpreter.mpFileDownloader->get(request);
     host.mLuaInterpreter.downloadMap.insert(reply, localFile);
+    connect(reply, &QNetworkReply::downloadProgress, [=](qint64 bytesDownloaded, qint64 totalBytes) {
+        raiseDownloadProgressEvent(L, urlString, bytesDownloaded, totalBytes);
+        TDebug(QColor(Qt::white), QColor(Qt::blue)) << "\n" << bytesDownloaded << "/" << totalBytes << " for file " << reply->url().toString() << "\n" >> 0;
```

Comment:
Other `TDebug` and `qDebug` calls are wrapped with `if (mudlet::debugMode)`  
I think this one should maybe get the same treatment or be removed since all the data is passed to the event.

Reply:
One need not have a Lua event handler coupled up to this event - so the Central Debug Console is a valid place to log it as well (provided it is on display...)

Reply:
Actually this was intention all along to add this `if` but at the last moment it slipped my mind.